docs/providers/aws/events/schedule.md

```diff
@@ -34,6 +34,8 @@ functions:
     handler: statistics.handler
     events:
       - schedule:
+          name: your-scheduled-rate-event-name
```

Comment:
I think it's better to leave existing examples as they are and add another one with name and description.